src/core_plugins/kibana/public/management/sections/objects/_view.html

```diff
@@ -1,36 +1,127 @@
-<kbn-management-app section="kibana">
-  <kbn-management-objects-view class="container">
-    <div class="pull-right" style="margin-top: 20px;">
-      <a href="{{ link }}" class="btn btn-primary">View {{ title }}</a>
-      <a confirm-click="delete()" class="btn btn-danger"><i class="fa fa-trash-o"></i> Delete {{ title }} Object</a>
+<kbn-management-app section="kibana" class="kuiView">
+  <kbn-management-objects-view class="kuiViewContent kuiViewContent--constrainedWidth">
+    <!-- Header -->
+    <div class="kuiViewContentItem kuiSubHeader">
+      <h1 class="kuiTitle">
+        Edit {{ title }}
+      </h1>
+
+      <div class="kuiButtonGroup">
+        <a
+          class="kuiButton kuiButton--basic kuiButton--iconText"
+          href="{{ link }}"
+        >
```

Comment:
One last objective argument for the dangling bracket is that it's easier to add/remove/shuffle attributes. You won't need to mess with the closing bracket to make sure it's on the last attribute.